libcxx/include/__math/traits.h

```diff
@@ -28,17 +28,23 @@ namespace __math {
 // signbit
 
 template <class _A1, __enable_if_t<is_floating_point<_A1>::value, int> = 0>
-_LIBCPP_NODISCARD inline _LIBCPP_HIDE_FROM_ABI bool signbit(_A1 __x) _NOEXCEPT {
+_LIBCPP_NODISCARD inline
```

Comment:
I think it would be better if all overloads have the same constexprness. I'd just define a `_LIBCPP_SIGNBIT_CONSTEXPR` for now.

Reply:
Great suggestion. Didn't think about it. Done.